sdk/insights/ci.yml

```diff
@@ -22,8 +22,8 @@ pr:
     include:
```

Comment:
While in here could you also update the header similar to what we did in https://github.com/Azure/azure-sdk-for-net/pull/14196/.